### ISSUE TITLE
Introduce `stable` git tag to prevent downstream breakage

### ISSUE THREAD

**Issue by cr** [mozmark/OneCRL-Tools#34]
#33 breaks downstream software. `go get` is unable to fetch specific git commits, but there's a web service that allows to install using specific tags from github repos.

```
git tag stable master~1  # or any other known-working commit
git push --tags
```

The idea then is to not replace that tag unless there is a newer commit that is *confirmed to be working well*.

**Comment by mozmark** [comment]
This should be OK now.

**Comment by jcjones** [comment]
I see it.

**Comment by cr** [comment]
Has been addressed in the course of #40
